src/core/tests/type_prop/is_nan.cpp

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2018-2022 Intel Corporation
```

Comment:
```suggestion
// Copyright (C) 2022 Intel Corporation
```
I've seen also this version 
Will this kind  legal comment prevent from update  it every year?

Reply:
I think that my version is a correct license statement.